CMakeLists.txt

```diff
@@ -215,6 +215,7 @@ option(THEROCK_ENABLE_MATH_LIBS "Enable building of math libraries" "${THEROCK_E
 option(THEROCK_ENABLE_ML_LIBS "Enable building of ML libraries" "${THEROCK_ENABLE_ALL}")
 option(THEROCK_ENABLE_PROFILER "Enable building the profiler libraries" "${THEROCK_ENABLE_ALL}")
 option(THEROCK_ENABLE_DC_TOOLS "Enable building of data center tools" "${THEROCK_ENABLE_ALL}")
+option(THEROCK_ENABLE_IREE_LIBS "Enable building of IREE libraries" "${THEROCK_ENABLE_ALL}")
```

Comment:
This should be changed to `OFF` before merge. Leaving it on temporarily just to show that CI _can_ pass. It will be added to the build once multi-arch support has landed, and hipDNN gets some breaking changes through (breaking changes are pretty inconvenient when working cross-repo).